src/main/scala/io/epiphanous/flinkrunner/model/sink/JdbcSinkConfig.scala

```diff
@@ -638,6 +624,20 @@ case class JdbcSinkConfig[ADT <: FlinkEvent](
     }
   }
 
+  def _matcher(value: Any) = {
+    val encoder = new JsonRowEncoder[Map[String, Any]]()
+    value match {
+      case ts: Instant => Timestamp.from(ts)
+      case m: Map[String, Any] => encoder.encode(m) match {
+        case Success(jsonString) => jsonString
+        case Failure(e) => {
+          throw new RuntimeException(s"Error occurred: ${e.getMessage}")
```

Comment:
A couple concerns here. First, the cosmetic one: I prefer we give a more specific message and pass the original error through, so: `throw new RuntimeException(s"Failed to json encode map $m", e)`.

A more important question is do we want to throw an error? This will kill the running job. Is that the behavior we want, or should we consider logging the error and returning null?

Reply:
select dbo.fnDateFormat('UN- UN- 2001', 'mm- dd- yyyy','eng')

Reply:
An error in one row can't stop the whole stream.

Reply:
@nextdude I was looking into using markers as a way to filter similar kids of errors in the streaming project - is that something that can work here as well?

Reply:
```
      override def open(config: Configuration): Unit = {
        this.mutationSuccess = getRuntimeContext.getMetricGroup.counter("csvParseSuccessCounter")
        this.mutationFailure = getRuntimeContext.getMetricGroup.counter("csvParseFailureCounter")
      }

      override def map(value: String): SomeCSVPOJOHolder = {
        try {
          val values = value.split(",")
          val timestamp = values(0).toLong
          val clazzInstance = SomeCSVPOJOHolder(values(0), values(1), values(2), values(3))
          mutationSuccess.inc()
          clazzInstance
        } catch {
          case _ => {
            this.mutationFailure.inc()
            println(s"Failure on value [$value]")
            null
          }
        }
      }
    }
```

Wanted to throw this out there too - this was a pattern that worked well for me in the past.